modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/SlimFrameworkServerCodegen.java

```diff
@@ -0,0 +1,195 @@
+package io.swagger.codegen.languages;
+
+import io.swagger.codegen.CodegenConfig;
+import io.swagger.codegen.CodegenConstants;
+import io.swagger.codegen.CodegenType;
+import io.swagger.codegen.DefaultCodegen;
+import io.swagger.codegen.SupportingFile;
+import io.swagger.models.properties.ArrayProperty;
+import io.swagger.models.properties.MapProperty;
+import io.swagger.models.properties.Property;
+
+import java.io.File;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.HashSet;
+
+public class SlimFrameworkServerCodegen extends DefaultCodegen implements CodegenConfig {
```

Comment:
Instead of extending `DefaultCodegen`, another approach is to extend PhpClientCodegen so as to leverage the typeMapping, languageSpecificPrimitives, etc that are already defined in PhpClientCodegen.

This is not to say you need to update the code to extend PhpClientCodegen but something worth sharing with you.